webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require("path");

```

Comment:
See the configuration of the working `webpack.config.js` found here https://github.com/AndcultureCode/AndcultureCode.JavaScript.Core/blob/main/webpack.config.js to address peer dependencies.  Might explain why you're having an issue.